lib/mixins/property-effects.js

```diff
@@ -2756,6 +2756,7 @@ export const PropertyEffects = dedupingMixin(superClass => {
           // once).
           const parent = template._parentTemplateInfo || this.__templateInfo;
           const previous = parent.lastChild;
+          templateInfo.parent = parent;
```

Comment:
It may not have had a template in `applyTemplateInfo`, so we always need to set the `parent` calculated on line 2757 to `templateInfo.parent` here.  So while we could potentially change `applyTemplateInfo`, we couldn't remove this code here. Seems not worth it.

Reply:
OK, makes sense.